enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/JWTAuthenticator.java

```diff
@@ -356,57 +367,44 @@ public int getPriority() {
     }
 
 
-
     /**
      * Validate scopes bound to the resource of the API being invoked against the scopes specified
      * in the JWT token payload.
      *
      * @param apiContext        API Context
      * @param apiVersion        API Version
-     * @param matchingResource  Accessed API resource
+     * @param matchingResources Accessed API resource
```

Comment:
```suggestion
     * @param matchingResources Accessed API resource or operations in GraphQL case
```

Reply:
Feel like it does not have to specifically mention gql apis here